### ISSUE TITLE
DHT11 not sending any humidity data

### ISSUE THREAD

**Issue by pleschev** [mysensors/NodeManager#489]
Setting up a basic NodeManager node with just a SensorDHT11 defined, humidity data isn't being sent from the sensor:

```
#include "sensors/SensorDHT11.h"
SensorDHT11 sensorDHT11(3);
```
Digging around the Nodemanager code:
https://github.com/mysensors/NodeManager/blob/master/sensors/SensorDHT.h#L54
I noticed that on the first call to onLoop() for the temperature, it all works correctly, on the second call of onLoop() for humidity, the sensor library returns NaN. I got it working by increasing the wait time from:
`nodeManager.sleepOrWait(_dht->getMinimumSamplingPeriod());
`
to:
`
nodeManager.sleepOrWait(_dht->getMinimumSamplingPeriod()+100);
`
This fixes the problem for me, however, I don't know what the best fix would be that is worthy of a pull request

**Comment by user2684** [comment]
Thanks for reporting this, tagging it as a bug. Yeah, DHT sensors are always tricky to pull from, adding this additional waiting time is definitely safer

**Comment by user2684** [comment]
Fixed with #510
